convert2rhel/checks.py

```diff
@@ -293,7 +310,7 @@ def get_unsupported_kmods(host_kmods, rhel_supported_kmods):
     Ignore certain kmods mentioned in the system configs. These kernel modules moved to kernel core, meaning that the
     functionality is retained and we would be incorrectly saying that the modules are not supported in RHEL."""
     return (
-        host_kmods
-        - rhel_supported_kmods
-        - set(system_info.kmods_to_ignore)
+            host_kmods
+            - rhel_supported_kmods
+            - set(system_info.kmods_to_ignore)
```

Comment:
Seems like IDE indented this too much

Reply:
Yeah.. PyCharm goes crazy, I'll investigate it..